contracts/connectors/loantoken/LoanToken.sol

```diff
@@ -10,20 +10,20 @@ import "./AdvancedTokenStorage.sol";
 
 contract LoanToken is AdvancedTokenStorage {
 
-    // It is important to maintain the variables order so the delegate calls can access sovrynContractAddress and wethTokenAddress
+    // It is important to maintain the variables order so the delegate calls can access sovrynContractAddress and wbtcTokenAddress
```

Comment:
should be `wrbtc` or is `wbtc`  alright?

Reply:
it is wrapped rBTC of course, but we try to give the user the impression he's working directly with BTC (although of course, he's working with rBTC). That's why I called it wBTC and not wRBTC. That's also why on the frontend rBTC was replaced with BTC. But it's a good point and worth getting a confirmation from Edan on the hidden core channel.